### ISSUE TITLE
Allow bulk configs to be overridable via mix tasks

### ISSUE THREAD

**Issue by imranismail** [infinitered/elasticsearch-elixir#26]
```
config :recommender, Recommender.Elasticsearch.Cluster,
  # When indexing data using the `mix elasticsearch.build` task,
  # control the data ingestion rate by raising or lowering the number
  # of items to send in each bulk request.
  bulk_page_size: 5000,

  # Likewise, wait a given period between posting pages to give
  # Elasticsearch time to catch up.
  # 15 seconds
  bulk_wait_interval: 15_000,
```

Something like this:

`mix elasticsearch.build users --cluster Recommender.Elasticsearch.Cluster --wait-interval 15000 --page-size 5000`

On top of that. Allowing it to be configured per index level would also be nice.

**Comment by imranismail** [comment]
Wow that was fast!

Thank you @danielberkompas

**Comment by danielberkompas** [comment]
No problem! I released `v0.4.0` with this change.
